bindings/nodejs/examples/1-create-account.js

```diff
@@ -28,4 +27,31 @@ async function run() {
     process.exit(0);
 }
 
+async function createAccountManager() {
+    const accountManagerOptions = {
+        storagePath: './alice-database',
+        clientOptions: {
+            nodes: [
+                {
+                    url: 'https://api.alphanet.iotaledger.net',
+                    auth: null,
+                    disabled: false,
+                },
```

Comment:
Not sure if you're aware, but should also be possible to just provide the url as String alone

Reply:
And this url shouldn't be there :trollface: